cmd/retrieval-client/main.go

```diff
@@ -55,21 +67,35 @@ func run(ctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	err = logging.SetLogLevel("main", "debug")
+	if err != nil {
+		return err
+	}
 
-	cidStr := ctx.String(queryFlag.Name)
+	cidStr := ctx.Args().First()
+	pieceCIDStr := ctx.String(pieceCIDFlag.Name)
 	bootnodesStr := ctx.String(bootnodesFlag.Name)
+	timeout := ctx.Float64(timeoutFlag.Name)
```

Comment:
convert this to seconds? it's currently in nanoseconds, i need to put a huge number for it to not timeout almost immediately